.github/workflows/main.yml

```diff
@@ -319,7 +320,7 @@ jobs:
         if [[ ! -d ./emsdk ]]; then
           git clone https://github.com/emscripten-core/emsdk.git
         else
-          bash -c "cd emsdk; git pull"
+          bash -c "cd emsdk; git fetch --all; git reset --hard origin/master"
```

Comment:
Did they not switch this one to `main`? They did it for the emscripten repo, but i don't know about emsdk. 

(Did I mention how much i fucking hate this branch-renaming braindeath?)

Reply:
They did, and it's even more awful because `origin/master` still resolves correctly, even though it's not valid. 😞

Reply:
It's "valid" in that it refers to an existing branch in the remote… they didn't bother deleting the old branch, and of course it's not being updated. Way to guarantee a smooth transition.

Reply:
Yep. It sucks.